src/main/java/io/deephaven/benchmark/api/BenchTable.java

```diff
@@ -232,13 +232,14 @@ private String replaceTableAndGeneratorFields(String query) {
 
         String codec = getCompression();
         codec = codec.equals("NONE") ? "UNCOMPRESSED" : codec;
+        codec = codec.equals("LZ4") ? "LZ4_RAW" : codec;
         String compression = String.format(", compression_codec_name='%s'", codec);
 
         return query.replace("${table.name}", tableName)
                 .replace("${compression.codec}", compression)
                 .replace("${max.dict.keys}", ", max_dictionary_keys=2000000")
-                .replace("${max.dict.bytes}", ", max_dictionary_size=20000000")
-                .replace("${target.page.bytes}", ", target_page_size=2000000")
+                .replace("${max.dict.bytes}", ", max_dictionary_size=16777216")
```

Comment:
There seems to be a difference between max_dictionary_keys here and in file ParquetTestRunner, similarly target page size. Can you please help me understand?

Reply:
This section of code sets the parquet settings for any benchmarks that do data generation using the Benchmark framework method.  This is completely separate from the Parquet benchmarks I've been showing you, which have their own settings and do their own data generation.

The reason I used bigger settings is because in testing I was able to get the best parquet read times with this using LZ4_Raw compression with a bigger page size.  Keep in mind that the majority of the standard benchmarks use int, float, and String for data types, not vectors/arrays or BigDecimals.

Reply:
Got it, thanks for explaining.